### ISSUE TITLE
Ability to hide all non-quest icons like traders, auction, etc with button

### ISSUE THREAD

**Issue by SegoItCh** [Questie/Questie#4140]
So sadly there is no button for this like for quests, so hard to see quests when everything I need enabled, and world map just flooded, would be cool to see this feature

**Comment by Cabro** [comment]
You can disable those at the Questie minimap button

**Comment by SegoItCh** [comment]
Only one by one, not all together

**Comment by SegoItCh** [comment]
Hide Questie minimap button not works for this icons too, can you fix it at least? My map just dead

**Comment by drejjmit** [comment]
By default only a few icons are enabled. Just uncheck the ones you don't want on the minimap icon or in ``/questie``. It will be saved and they will stay off on that character. Should take no more than 1 minute.

**Comment by SegoItCh** [comment]
So I have to do this every time I want look at world map?

**Comment by drejjmit** [comment]
So you want to temporary hide them? Not turn them off completely?

**Comment by SegoItCh** [comment]
Yes, sometime I want see quests, sometime vendors, etc, but they overlap, sometime I want see clear map
